### ISSUE TITLE
Click on link (except the first one) which opens a new tab executed 5+ seconds

### ISSUE THREAD

**Issue by DudaGod** [SeleniumHQ/selenium#6344]
## Meta -
OS:  Linux/Mac os
Selenium Version:   3.12.0
Browser:  Chrome
Browser Version: 67.0.3396.62 and 70.0.3535.0

## Detailed information
I ran into a problem when running my test in chrome.
In the test I clicked links one after another which opens a new tab.
And the strange thing that each subsequent click (except first) is performed 5+ seconds.
My test code - https://pastebin.com/adiBqGNk.
Log from my console:
```
Execution time (hr): 0s 616.269329ms
Execution time (hr): 5s 429.178605ms
Execution time (hr): 5s 492.508944ms
```

Log from selenium - https://pastebin.com/DqSZ2wj7
And a piece from the selenium log with a pause of 5 seconds looks like:
```
2018-08-28T01:43:20.306510207Z [1535420600.306][DEBUG]: DEVTOOLS COMMAND Input.dispatchMouseEvent (id=46) {
2018-08-28T01:43:20.306514515Z    "button": "none",
2018-08-28T01:43:20.306518706Z    "clickCount": 0,
2018-08-28T01:43:20.306534622Z    "modifiers": 0,
2018-08-28T01:43:20.306538269Z    "type": "mouseMoved",
2018-08-28T01:43:20.306554021Z    "x": 210,
2018-08-28T01:43:20.306558541Z    "y": 488
2018-08-28T01:43:20.306562380Z }
2018-08-28T01:43:25.309820104Z [1535420605.310][DEBUG]: DEVTOOLS RESPONSE Input.dispatchMouseEvent (id=46) {
```
Video of test execution - https://youtu.be/cuYphdDuBq0

## What is the expected behavior?
I expect that none of the clicks will hang for 5+ seconds

## Did this work before?
Yes. It works in chromium@59, but in chromium@60+ it does not.

## Does this work in other browsers?
Yes. It works correctly (much faster) in firefox, edge, ...

**Comment by GitSage** [comment]
I was able to reproduce your problem too. I was able to get around it by switching back to the first tab before clicking the link again. I rewrote your example without `webdriverio` to rule that out as a possible issue.

```
'use strict';

const {Builder, By} = require('selenium-webdriver');

let driver;
(async () => {
	driver = await new Builder()
		.forBrowser('chrome')
		.build();

	await driver.get('https://search.yahoo.com/search?p=foobar');
	await openNewTab(By.css('.searchCenterMiddle li:nth-of-type(1) .title'));
	await openNewTab(By.css('.searchCenterMiddle li:nth-of-type(2) .title'));
	await openNewTab(By.css('.searchCenterMiddle li:nth-of-type(3) .title'));

	await driver.quit();
})();


async function openNewTab(selector) {
	let hrstart = process.hrtime();
	await driver.findElement(selector).click();
	let hrend = process.hrtime(hrstart);
	console.log("Execution time (hr): %ds %dms", hrend[0], hrend[1] / 1000000);

	// switch back to first tab
	const allTabs = await driver.getAllWindowHandles();
	await driver.switchTo().window(allTabs[0]);
}
```

```
Execution time (hr): 0s 150.291665ms
Execution time (hr): 0s 176.316139ms
Execution time (hr): 0s 200.823576ms
```

**Comment by DudaGod** [comment]
Yeah, I also found this solution. But it's only fast workaround and it will probably help the chrome developers find the problem faster.

But I do not want to perform 2 additional commands in all my tests. I want to return the behaviour from chrome@59 in which click was done quickly.

I've tried to disable all flags in `chrome://flags`, set all possible capabilitites and commands for chromedriver from https://peter.sh/experiments/chromium-command-line-switches/. 
Nothing helped =( 

I even tried to build my custom version of chromium on a certain tag to find in which one it broke.
But in my mac it is building very long time and in view of the fact that versions of chromium@60 100+ I will find the broken tag through a fairly long time.

Anyway thank you man, I appreciate your investigation.
